pom.xml

```diff
@@ -9,12 +9,15 @@
 
   <modules>
     <module>robotpy</module>
-    <module>robotpy.feature</module>
+    
     <module>robotpy.updatesite</module>
+    
+    
+    <module>robotpy.feature</module>
```

Comment:
Hm? Why did this get moved around?

Reply:
I think that was just an eclipse being odd thing. I didnt manually do that.